guides/common/assembly_integrating-project-with-keycloak.adoc

```diff
@@ -0,0 +1,11 @@
+[id="integrating-project-with-hat-keycloak-for-external-authentication_{context}"]
+= Integrating {Project} with {Keycloak} for External Authentication
+
+You can configure {Project} to use {Keycloak} as an external authentication source.
+
+You can also configure {Keycloak} to use CAC cards or Time-based One-time Password (TOTP) for authentication.
```

Comment:
These are the only ways as of now which are supported for authentication, might as well remove the `also`, what do you think?